locator/topology.hh

```diff
@@ -348,6 +352,10 @@ public:
 
     void for_each_node(std::function<void(const node*)> func) const;
 
+    std::unordered_set<const node*> get_nodes() const;
```

Comment:
Comment that it skips "none" and "left" may be useful.

Reply:
OK, I'll add the same comment to `for_each_node`.